Delete the simple-yaml.md example

feihujiang opened the pull request:
Fix #15355

feihujiang committed:
Delete the simple-yaml.md example

bgrant0607 commented:
Thanks. 

Is replication.yaml tested in examples/examples_test.go?

Please search for links to this doc and update them also.

feihujiang commented:
No, the replication.yaml tested isn't in examples/examples_test.go.
The only replication.yaml I searched is in user-guide/managing-deployments.md, it is part of URL, "https://raw.githubusercontent.com/GoogleCloudPlatform/kubernetes/master/docs/user-guide/replication.yaml". and I changed.
I searched for links to this doc and updated them to deploying-applications.md.

feihujiang commented:
@bgrant0607 Could you please review this PR?

ashcrow commented:
Shippable failure looks like a flake:

```
k8s.io/kubernetes/pkg/storage/etcd	2.692s
```

bgrant0607 commented:
cc @jeffmendoza @amygdala

janetkuo commented on docs/admin/daemons.md:
```diff
@@ -77,7 +77,7 @@ but with different flags and/or different memory and cpu requests for different
 ### Required Fields
 
 As with all other Kubernetes config, a DaemonSet needs `apiVersion`, `kind`, and `metadata` fields.  For
-general information about working with config files, see [here](../user-guide/simple-yaml.md),
```
Can you change each "here" to the title of individual links?

janetkuo commented on docs/admin/daemons.md:
```diff
@@ -77,7 +77,7 @@ but with different flags and/or different memory and cpu requests for different
 
 As with all other Kubernetes config, a DaemonSet needs `apiVersion`, `kind`, and `metadata` fields.  For
-general information about working with config files, see [here](../user-guide/simple-yaml.md),
+general information about working with config files, see [here](../user-guide/deploying-applications.md),
```
Can you change each "here" to the title of individual links?

janetkuo commented on docs/user-guide/ingress.md:
```diff
@@ -105,7 +105,7 @@ A minimal Ingress might look like:
 *POSTing this to the API server will have no effect if you have not configured an [Ingress controller](#ingress-controllers).*
 
-__Lines 1-4__: As with all other Kubernetes config, an Ingress needs `apiVersion`, `kind`, and `metadata` fields.  For general information about working with config files, see [here](simple-yaml.md), [here](configuring-containers.md), and [here](working-with-resources.md).
+__Lines 1-4__: As with all other Kubernetes config, an Ingress needs `apiVersion`, `kind`, and `metadata` fields.  For general information about working with config files, see [here](deploying-applications.md), [here](configuring-containers.md), and [here](working-with-resources.md).
```
ditto

janetkuo commented on docs/user-guide/jobs.md:
```diff
@@ -143,7 +143,7 @@ $ kubectl logs pi-aiw0a
 
 As with all other Kubernetes config, a Job needs `apiVersion`, `kind`, and `metadata` fields.  For
-general information about working with config files, see [here](simple-yaml.md),
+general information about working with config files, see [here](deploying-applications.md),
```
ditto

janetkuo commented on docs/user-guide/managing-deployments.md:
````diff
@@ -117,8 +117,8 @@ It is a recommended practice to put resources related to the same microservice o
 ```console
-$ kubectl create -f https://raw.githubusercontent.com/GoogleCloudPlatform/kubernetes/master/docs/user-guide/replication.yaml
-replicationcontrollers/nginx
+$ kubectl create -f https://raw.githubusercontent.com/GoogleCloudPlatform/kubernetes/master/docs/user-guide/pod.yaml
````
Why change this?

feihujiang replied:
Because  docs/user-guide/replication.yaml will be deleted by this PR.  And Changing the URL doesn't change the meaning of the example.

feihujiang commented:
@janetkuo I updated this PR based on your feedback, please review again.

janetkuo commented on docs/admin/daemons.md:
```diff
@@ -77,8 +77,8 @@ but with different flags and/or different memory and cpu requests for different
-general information about working with config files, see [here](../user-guide/simple-yaml.md),
-[here](../user-guide/configuring-containers.md), and [here](../user-guide/working-with-resources.md).
+general information about working with config files, see [deploying applications](../user-guide/deploying-applications.md),
+[configuring containers](../user-guide/configuring-containers.md), and [working with resources](../user-guide/working-with-resources.md).
```
add "documents" to the end

janetkuo commented on docs/user-guide/simple-nginx.md:
```diff
@@ -84,7 +84,7 @@ In order to access your nginx landing page, you also have to make sure that traf
 ### Next: Configuration files
 
-Most people will eventually want to use declarative configuration files for creating/modifying their applications.  A [simplified introduction](simple-yaml.md)
+Most people will eventually want to use declarative configuration files for creating/modifying their applications.  A [deploying applications](deploying-applications.md)
```
"A deploying applications" isn't right; keep "A simplified introduction" or use more appropriate description

feihujiang commented:
@janetkuo Updated based on your advices, please have a look.

feihujiang commented:
@k8s-bot test this please

ashcrow commented:
@k8s-teamcity-mesosphere error is a known flake. If you want it to pass you can rebase and it should go away.

feihujiang commented:
@ashcrow Thanks for providing such information. I can ignore them :)

ashcrow commented:
Looks good to me!

janetkuo commented:
Applying lgtm since we can ignore the mesos smoke test failure.
